pom.xml

```diff
@@ -65,6 +69,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>io.quarkus.qe</groupId>
+                <artifactId>quarkus-test-openshift</artifactId>
+                <version>${quarkus.qe.framework.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
This dependency was already defined under the dependencies section (as far I see there is no needed to be under the dependenciesManagement section).